web/public/js/directives/cd-eventbrite-integration.js

```diff
@@ -0,0 +1,76 @@
+;(function() {
+  'use strict';
+
+function cdEventbriteIntegration($window, cdEventbriteService, alertService, atomicNotifyService,
+  $localStorage, $stateParams, $state, $translate, $rootScope) {
+    return {
+      restrict: 'E',
+      template:
+        '<i class="fa fa-spinner" ng-show="saving"></i>' +
+        '<div class="row cd-input-row" ng-hide="saving">' +
+          '<label class="col-lg-4 cd-form-label1">EventBrite</label>' +
+          '<div class="col-lg-8">' +
+            '<button type="button" class="btn btn-default" ' +
+            'name="dojoEventbrite" id="dojoEventbrite" ' +
+            'ng-click="authorizeOAuthEventBrite()">{{eventbriteText}}</button>' +
+          '</div>' +
+        '</div>',
+      controller: function($scope){
```

Comment:
Why inject $scope? Could just use `bindToController` instead. Or even write the whole thing as a component instead of a directive. Also, if kept this way, use array syntax for minification support.